src/widgets.ts

```diff
@@ -84,8 +106,20 @@ export class BokehView extends DOMWidgetView {
 
   protected _change_event(event: DocumentChangedEvent): void {
     const {ModelChangedEvent} = bk_require("document/events")
-    if (!this._blocked && event instanceof ModelChangedEvent)
-      this.send({event: "jsevent", id: event.model.id, new: event.new_, attr: event.attr, old: event.old})
+    if (!this._blocked && event instanceof ModelChangedEvent) {
```

Comment:
Can you add an overall comment explaining what the intent and general design is?  It looks like "only push a new event if idle"?  Whatever it is, please explain it, so that a reader can evaluate if the code measures up to the intent and can someday improve it if need b.

Reply:
Sure, once I'm actually happy with the implementation.